contents/preface.md

```diff
@@ -22,6 +22,7 @@ We make an argument for **incorporating software engineering practices into data
 Most data analyses are collaborative endeavors; that is why these software practices will help you.
 
 ### Data is Everywhere {#sec:data_everywhere}
+<!-- NOTE: make it an unnumbered section (there is no section 1.0.2) -->
```

Comment:
Dear authors,  
in case I should have more of these "suggestions" for the remaining chapters of the book, shall I write them as shown here or add one or more issues instead?

Reply:
```suggestion
```